### PR TITLE
Update specification for Engineering Schematic

### LINKED ISSUE #14: Add specification for the system schematic generation
This is to update the specification with the requirements pertaining to the generation of the system schematic.

### PR DESCRIPTION
**Pull request by AntoineGautier**
This closes #14.
If needed, 

- the HTML version of the specification is available online on my private repo for now, see https://antoinegautier.github.io/linkage.js/requirements.html#engineering-schematic,
- the PDF version is available at https://github.com/lbl-srg/linkage.js/blob/issue14_schematic/specification/build/latex/Linkage.pdf,
- the recording of the technical meeting is available at https://drive.google.com/file/d/14lHe_pPgqa0VTh0kFgUKsi8X_1AX_Tj2/view?usp=sharing.

### CODE REVIEW

File: specification/source/requirements.rst [hunk 1]
```diff
@@ -610,48 +610,112 @@ Documentation Export
...
-* an index tag is then generated based on the object position, from top to bottom and left to right,
-* eventually connection lines are drawn to link those tags to the four different control points buses (AI, AO, DI, DO). The line must be vertical, with an optional horizontal offset from the index tag to avoid overlapping any other object.
+  defined in the parent class ``Templates.AirHandlersFans.Components.OutdoorReliefReturnSection.Interfaces.PartialOutdoorReliefReturnSection``, where some parameters are assigned within the derived class ``Templates.AirHandlersFans.Components.OutdoorReliefReturnSection.Economizer`` as below.
 
-SVG is the required output format.
+  .. code-block::
 
-See :numref:`screen_schematics_output` for the typical output of the schematics export process.
+     model Economizer "Air economizer"
+       extends Buildings.Templates.AirHandlersFans.Components.OutdoorReliefReturnSection.Interfaces. PartialOutdoorReliefReturnSection(
```
Comment (mwetter) [inline specification/source/requirements.rst:688]: Remove the empty space in front of 'PartialOutdoorReliefReturnSection'

File: specification/source/requirements.rst [hunk 2]
```diff
@@ -610,48 +610,112 @@ Documentation Export
...
 
-Engineering Schematics
+Engineering Schematic
 ======================
 
-Objects of the original model to be included in the schematics export must have a declaration annotation providing the SVG file path for the corresponding engineering symbol e.g. ``annotation(__Linkage(symbol_path="value of symbol_path"))``.
+SVG is the required output format, DXF or DWG is optional.
+
+It is expected that Linkage will eventually be used to generate design documents included in the invitation to tender for HVAC control systems.
+As such, the exported graphics should meet the industry standards and have a pixel perfect precision.
```
Comment (mwetter) [inline specification/source/requirements.rst:635]: "pixel perfect" is not clear. Do you mean should be vector graphic? Or that lines need to meet each other?

**Review by mwetter — APPROVED** [review]
The specification looks good. Please see inline comments and the two items below for minor feedback. From my point of view, this is fine to merge.

Should the icon area be expanded to include the svg diagram? For example, in `Buildings/Templates/Components/Coils/Interfaces/PartialCoil.mo` the svg diagram is outside the icon area.

This comment is more related to the template rather than the specification: The graphic shows a dp measurement for where the filter is. Should there be a component for a pressure drop of the filter (we could move `dpSup`), and a Modelica dp model? Similarly, the outdoor air intake has a dp measurement but no Modelica component. When we get to the point of generating point lists, these are points that need to be trended, and if the model where to be used as a digital twin, it would be better to have a Modelica output for this quantity. See below for the graphic, which is from `DedicatedDamperPressure.mo`:
image

File: specification/source/requirements.rst (hunk 2 above)
Comment (AntoineGautier) [reply specification/source/requirements.rst:635]: I propose to replace with:

> shape edges, connections between objects, alignment or centering of objects, etc. should be precise at pixel level.

Comment (mwetter) [reply specification/source/requirements.rst]: That sounds good and is clear.

**Comment by AntoineGautier** [comment]
@WatchMeCalculate Is there anything blocking on your end or may I go ahead and merge that update?

**Comment by AntoineGautier** [comment]
Merging this PR as Elaine (Devetry) notified that it was validated by Jake on 1/12/22.

> Antoine, I have added it to the agenda, but Jake has looked at it and these changes are approved by us.  Thank you for pointing this out and checking with us, it's appreciated!

**Pull request merged**